app/views/collection_drafts/new.html.erb

```diff
@@ -5,14 +5,20 @@
 <% end %>
 
 <% if get_resource.errors.any? %>
-  <div id="error_explanation">
-    <h2><%= pluralize(get_resource.errors.count, 'error') %>) prohibit this draft from being saved:</h2>
-    <ul>
-      <% get_resource.errors.full_messages.each do |msg| %>
-        <li><%= msg %></li>
-      <% end %>
-    </ul>
-  </div>
+  <section class="errors">
```

Comment:
When considering what we want to try to reuse in partials, I think this is a good candidate.

Reply:
sounds good. feel free to do that either now or in the next ticket.